### PR TITLE
[fix][doc] Modify comments and doc for brokerDeleteInactiveTopicsMaxInactiveDurationSeconds

### PR DESCRIPTION
**Pull request by futeng**
### Motivation

Remove "not", because configuring the value for the parameter brokerDeleteInactiveTopicsMaxInactiveDurationSeconds will allow the deletion to take effect, not the other way around.

### Modifications

Simply remove the "not" from all relevant Java files and Markdown documents

### Documentation
 
- [x] `doc` 

- [ ] `doc-complete`

(I did keyword searches and substitutions, but wasn't sure if I missed anything.)

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@futeng thank you!
1. For each PR, only choose one doc label. Label guideline: https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0/edit#bookmark=id.idby085lkv0t
2. Please @ me in a comment if you need my review, or else I might miss it.

**Comment by Anonymitaet** [comment]
@horizonzy 
could you please review this PR from the technical perspective? Thank you!

File: conf/broker.conf [hunk 1]
```diff
@@ -188,7 +188,7 @@ brokerDeleteInactiveTopicsMode=delete_when_no_subscriptions
 brokerDeleteInactivePartitionedTopicMetadataEnabled=false
 
 # Max duration of topic inactivity in seconds, default is not present
-# If not present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used
```
Comment (horizonzy) [inline conf/broker.conf:191]: Should not change it

File: deployment/terraform-ansible/templates/broker.conf [hunk 2]
```diff
@@ -133,7 +133,7 @@ brokerDeleteInactiveTopicsFrequencySeconds=60
 brokerDeleteInactiveTopicsMode=delete_when_no_subscriptions
 
 # Max duration of topic inactivity in seconds, default is not present
-# If not present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used
+# If present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used
```
Comment (horizonzy) [inline deployment/terraform-ansible/templates/broker.conf:136]: Should not change it

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 3]
```diff
@@ -576,7 +576,7 @@ public class ServiceConfiguration implements PulsarConfiguration {
     @FieldContext(
         category = CATEGORY_POLICIES,
         doc = "Max duration of topic inactivity in seconds, default is not present\n"
-        + "If not present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used\n"
+        + "If present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used\n"
```
Comment (horizonzy) [inline pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java:579]: Should not change it

File: conf/broker.conf (hunk 1 above)
Comment (horizonzy) [reply conf/broker.conf:191]: https://github.com/apache/pulsar/blob/f230d15ffcd5f74cca13bd23b35ace784d6f8ce6/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2834-L2840

**Comment by horizonzy** [comment]
From source code, the doc is correct. Why do you think the doc is not match the behavior, give me more context.
@futeng

**Comment by futeng** [comment]
@horizonzy @Anonymitaet 
Sorry, I confused the parameters of the comment. I think the original comment is correct and needs no modification.
Sorry for wasting your time. I'll close this PR.

**Comment by Anonymitaet** [comment]
@futeng no worries. Welcome more contributions and discussions! 👏

**Pull request closed without merging**